app/controllers/dashboards_controller.rb

```diff
@@ -22,9 +31,29 @@ def export
     redirect_to dashboards_path, alert: e.message
   end
 
+  def push
+    version = Version.preview_version
+
+    if version.blank?
+      flash.alert = 'No preview version available'
+    else
+      pv = Version.create(number: version.number, production: true, user: current_user)
+
+      if pv.persisted?
+        flash.notice = 'Production data updated'
+      else
+
+        flash.alert = "Production data not updated: #{version.inspect} #{pv.errors.inspect}"
+      end
+    end
+
+    redirect_to dashboards_path
+  end
+
   private
 
   def csv_model(csv_type)
+    return Institution if csv_type == 'Institution'
```

Comment:
I think we're going to want specs around both of these just in case something changes. It might be a pain in the ass to write it as request specs, so this could be a code smell. Possibly consider putting this in a model that you can test at unit level.